autoscheduler/frontend/src/components/NavBar.tsx

```diff
@@ -23,6 +24,51 @@ const useStyles = makeStyles((theme) => ({
 const NavBar: React.SFC = () => {
   const classes = useStyles(appTheme);
 
+  const [usersName, setUsersName] = React.useState('');
+  React.useEffect(() => {
```

Comment:
Add a comment explaining that what this does (check if the user is logged in and sets name if so)